sheoranharis.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "Harish Sheoran",
+    "type": "personal",
+    "bio": "Pythonista",
+    "avatar": "https://github.com/sheoranharis.png",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/sheoranharis",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/sheoranharis",
+            "icon": "twitter"
+        }
+      {
+            "name": "Read my Blog ",
+            "url": " https://hashnode.com/@sheoranharis",
+            "icon": " Hashnode"
```

```suggestion
            "url": "https://hashnode.com/@sheoranharis",
            "icon": "Hashnode"
```
Remove these extra spaces :point_up:

---

ok,
done